gtda/time_series/embedding.py

```diff
@@ -208,52 +213,169 @@ def plot(Xt, sample=0, plotly_params=None):
         return plot_point_cloud(Xt[sample], plotly_params=plotly_params)
 
 
+def time_delay_embedding(
+        X, time_delay=1, dimension=2, stride=1, flatten=False,
+        ensure_last_value=True, validate=True
+        ):
```

Comment:
The linting rules are being changed (automatically, I believe) on the files that you modify. I am not very enthusiastic at having 4 lines of code when there could be only 2, with no real increase in readability.

Reply:
I'm happy to reduce such occurrences!